clients/roscpp/.catkin_workspace

```diff
@@ -0,0 +1 @@
+# This file currently only serves to mark the location of a catkin workspace for tool integration
```

Comment:
Please remove this file.

Reply:
I have removed it. Sorry for missing it.